projects/systelab-components/src/lib/combobox/autocomplete/autocomplete-api-combobox.component.spec.ts

```diff
@@ -212,4 +213,14 @@ describe('AutocompleteApiAutocomplete', () => {
 		component.combobox.onCellKeyDown(event);
 		expect(component.combobox.inputElement.nativeElement.value).toEqual('aa');
 	});
+
+	it('onEnterDoSelect', () => {
+		const selectThisNodeSpy = spyOn(RowNode.prototype, 'selectThisNode');
+		component.combobox.onEnterDoSelect(new KeyboardEvent('keydown', {}));
+		if (component.combobox.isDropdownOpened) {
```

Comment:
set the isDropdownOpened a false and check that the method is not been called and other it setting to true and checking is called